torch/_dynamo/config.py

```diff
@@ -42,8 +42,8 @@
 # disable (for a function) when cache reaches this size
 cache_size_limit = 64
 
-# specializing int/float by default
-specialize_int_float = True
+# specializing int by default?
+specialize_int = False
```

Comment:
I decided that I am only going to unspecialize ints, and NOT floats. This side-steps the discussion of whether or not float inputs should be repped as 0d tensors or SymFloat.